core/templates/pages/exploration-player-page/layout-directives/audio-bar.component.ts

```diff
@@ -69,15 +64,6 @@ export class AudioBarComponent {
     this.selectedLanguage = {
       value: this.audioTranslationLanguageService.getCurrentAudioLanguageCode()
     };
-
```

Comment:
Removed unused code.
Since the audio bar is working fine without these lines.
Deferring @ashutoshc8101 to cross-check once.
Thanks!